src/test/java/teammates/test/cases/search/FeedbackResponseCommentSearchTest.java

```diff
@@ -25,8 +25,9 @@ public void allTests() {
                 .get("comment1FromT1C1ToR1Q1S1C1");
         FeedbackResponseCommentAttributes frc1I1Q2S1C1 = dataBundle.feedbackResponseComments
                 .get("comment1FromT1C1ToR1Q2S1C1");
-        FeedbackResponseCommentAttributes frc1I3Q1S1C2 = dataBundle.feedbackResponseComments
-                .get("comment1FromT1C1ToR1Q1S1C2");
+        //Modify it after discussing, see if the build passes
+        /*FeedbackResponseCommentAttributes frc1I3Q1S1C2 = dataBundle.feedbackResponseComments
```

Comment:
@whipermr5 Want to clarify something. The tests that I have commented out are dealing with the `Private` session that is in `typicalDataBundle.json` file. But the tests are not `Private session` related itself. Should I keep them or should I remove them, as the private session data in `typicalDataBundle.json` has already been deleted.

Reply:
@whipermr5 Used the other session in that course `Instructor feedback session` instead, modified the private session's question,comment,response slightly changed it to `Instructor feedback session`'s questioin,comment and response.